packages/main/test/pages/Toolbar.html

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+
+	<title>Toolbar</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<meta charset="utf-8">
+
+	<script src="../../bundle.esm.js" type="module"></script>
+
+	<script>// delete Document.prototype.adoptedStyleSheets;</script>
+	<style>
```

Comment:
Usually we define page styles should be placed in `packages/main/test/pages/styles`